apps/web/pages/_app.tsx

```diff
@@ -29,6 +29,9 @@ function MyApp(props: AppProps) {
   } else if (router.pathname === "/500") {
     pageStatus = "500";
   }
+  // Use the layout defined at the page level, if available
+  const getLayout = Component.getLayout ?? ((page) => page);
+
```

Comment:
This will allow us to set reusable layouts at page level

Reply:
super cool, this is new, right?

Reply:
New in Next? No. New for our codebase? Yes